### PR TITLE
sql: update set variables

### PR DESCRIPTION
**Pull request by jackysp**

### CODE REVIEW

**Comment by tiancaiamao** [comment]
LGTM

File: sql/variable.md [hunk 1]
```diff
@@ -32,6 +32,70 @@ SET @@autocommit = 1;
 
 * `LOCAL` 以及 `@@local.` 是 `SESSION` 以及 `@@session.` 的同义词
 
+### 系统变量的作用机制
+
+* 会话范围的系统变量仅仅会在创建会话时才会根据全局范围系统变量初始化自己的值。更改全局范围的系统变量不会改变已经创建的会话正在使用的系统变量的值。
+
+```sql
```
Comment (yikeke) [inline sql/variable.md]: Please indent this code block with 4 spaces.

Comment (yikeke) [reply sql/variable.md]: In this variable.md file, there are two other code blocks that need to be indented with 4 spaces, plz indent them as well.

Comment (jackysp) [reply sql/variable.md]: PTAL @yikeke

**Pull request merged**